test/e2e_node/eviction_test.go

```diff
@@ -1083,7 +1149,7 @@ func podWithCommand(volumeSource *v1.VolumeSource, resources v1.ResourceRequirem
 					Command: []string{
 						"sh",
 						"-c",
-						fmt.Sprintf("i=0; while [ $i -lt %d ]; do %s i=$(($i+1)); done; while true; do sleep 5; done", iterations, command),
+						fmt.Sprintf("i=0; while [ $i -lt %d ]; do %s i=$(($i+1)); done; while %v; do sleep 5; done", iterations, command, sleepAfterExecuting),
```

Comment:
I recently learned that not trapping `TERM` here causes pause deletion to take a lot longer. We have a utility for that:

```go
cmd := fmt.Sprintf("i=0; while [ $i -lt %d ]; do %s i=$(($i+1)); done", iterations, command)
if sleepAfterExecuting {
	cmd += "; " + e2epod.InfiniteSleepCommand
}
```

Reply:
Done, thank you.